rts/lra/trip-controller/pom.xml

```diff
@@ -89,9 +103,17 @@
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-client</artifactId>
-            <version>${version.resteasy}</version>
-            <scope>test</scope>
+            <scope>compile</scope>
         </dependency>
+        <dependency>
+    		<groupId>commons-logging</groupId>
+    		<artifactId>commons-logging</artifactId>
+    		<version>1.2</version>
+		</dependency>
```

Comment:
@marcosgopen FYI I there are a couple of misaligned </dependency> in this file but I don't think it should block it merging so I will approve the request. If you would like to fix it, please do re-request a review but otherwise I am good with it being merged. Thank you!